scripts/phantomas_dwis

```diff
@@ -229,8 +229,7 @@ gradient_theta = np.arccos(gradient_directions[:, 2])
 gradient_phi = np.arctan2(gradient_directions[:, 1], gradient_directions[:, 0])
 H = shm.matrix(gradient_theta, gradient_phi, order=order_sh)
 for bval in unique_bvals:
-    qnorm = np.sqrt(bval / (4 * np.pi**2 * tau))
-    rs = model.signal_convolution_sh(order_sh, qnorm, tau)
+    rs = model.signal_convolution_sh(order_sh, bval)
```

Comment:
Great, and I think you could also remove the line defining `tau`, which is no longer relevant (a few lines above).